api/serverless.yml

```diff
@@ -61,6 +61,9 @@ provider:
         - Effect: 'Allow'
           Resource: 'arn:aws:s3:::science-octopus-publishing-images-${self:provider.stage}/*'
           Action: 's3:PutObject'
+        - Effect: 'Allow'
```

Comment:
looks fine but might have been able to use a pattern in Resource but lets leave for another day...